core/src/App.html

```diff
@@ -604,14 +604,37 @@
         });
       },
       showAlert(settings, openFromClient = false) {
+        const currentAlerts = this.get().alerts;
+        if (
+          settings.id &&
+          currentAlerts &&
+          currentAlerts.find(a => a.settings.id === settings.id)
+        ) {
+          console.error(
+            `The alert with id '${
+              settings.id
+            }' already exists in a queue, therefore it won't be displayed `
+          );
+          return Promise.reject();
+        }
+
+        if (settings.closeAfter) {
+          setTimeout(() => {
```

Comment:
If the alert gets closed manually before the timeout an error is shown in the console: "An unexisting alert has been dismissed.". There is nothing wrong with closing an alert manually so there should not be any error shown.

Reply:
I'll investigate it.